content/about/clubs.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Affiliated Clubs
+layout: single
+# Card images should all be 348x196
+---
+
+The UBC CSSS is affiliated with the following UBC CS-related clubs.
```

Comment:
I guess same idea as above, we should probably say `The following UBC CS-related clubs are partners of the CSSS.`

Reply:
Will do!